http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -150,12 +150,13 @@ else
 fi
 
 if grep -qF "gcr.io" <<< "${JUPYTER_DOCKER_IMAGE}${RSTUDIO_DOCKER_IMAGE}${PROXY_DOCKER_IMAGE}${WELDER_DOCKER_IMAGE}" ; then
-  log 'Authorizing GCR...'
-  DOCKER_COMPOSE="docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /var:/var -w=/var cryptopants/docker-compose-gcr"
-else
-  DOCKER_COMPOSE="docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /var:/var docker/compose:1.29.2"
+  log 'Authorizing GAR...'
+  $GCLOUD_CMD auth configure-docker
```

Comment:
Not sure how important this is, but I think the reason `cryptopants/docker-compose-gcr` was used was to support private images. The [cryptopants/docker-compose-gcr](https://hub.docker.com/r/cryptopants/docker-compose-gcr) doc says:
```
This container helps resolve an issue with Container Optimized OS. The OS does not come with docker-compose, and instead expects users to use containerized docker-compose. However the official containerized docker-compose does not include docker-credential-gcr, which is needed to pull Docker images from a private Google Container Registry.

This container uses the official docker-compose as a base, and adds docker-credential-gcr. It can be used just like the normal containerized docker-compose, but can now access gcr.io.
```

Do private images still work after this change?

(All our standard images are public, but technically we do support private GCR images. I'm not sure if there are any users currently using this feature though.)

Reply:
ahh ok, I just tested that with an image not in the broad public repo and didn't work on this branch or on dev/prod, so it seems like something we may have supported in the past but now isn't working...

Reply:
I can try to fix it though